odl/phantom/noise.py

```diff
@@ -25,17 +25,62 @@
 import numpy as np
 
 
-__all__ = ('white_noise',)
+__all__ = ('white_noise', 'poisson_noise')
 
 
 def white_noise(space):
     """Standard gaussian noise in space, pointwise N(0, 1).
 
+    Parameters
+    ----------
+    space : `FnBase`
+        The space that the noise should be created in
+
+    Returns
+    -------
+    white_noise : ``space`` element
+
+    See Also
+    --------
+    poisson_noise
+    numpy.random.randn
     """
     values = np.random.randn(*space.shape)
     return space.element(values)
 
 
+def poisson_noise(intensity):
+    """Poisson distributed noise with given intensity.
+
+    Parameters
+    ----------
+    intensity : `FnBase`
+        The intensity (usually called lambda) parameter of the noise.
+
+    Returns
+    -------
+    poisson_noise : ``intensity.space`` element
+        Poisson distributed random variable.
+
+    Notes
+    -----
+    The probability density function for value :math:`k` and intensity
```

Comment:
Weird formulation. Why speak of a function when it is a discrete probability measure? My suggestion is to use a phrasing like 
```
For a Poisson distributed random variable :math:`X` with intensity
:math:`\\lambda`, the probability of it taking the value 
:math:`k \\in \mathbb{N}_0` is given by

    .. math::
        P(X = k) = \\frac ...
```